packages/plugin-xml/src/Attributes.ts

```diff
@@ -6,7 +6,7 @@ export class Attributes extends Modifier {
     private _record: Record<string, string> = {};
     style = new CssStyle();
     classList = new ClassList();
-    constructor(attributes?: Record<string, string> | Attributes | NamedNodeMap) {
+    constructor(attributes?: Attributes | NamedNodeMap) {
```

Comment:
So I'm not sure I should have removed this (see below).

Reply:
Mouais, je suis pas super convaincu mais si tu veux le remettre I won't fight for it. Par contre le met pas comme premier type si c'est un truc qu'on n'utilise pas vraiment mets le plutot à la fin.